packages/common/src/discussions/_internal/DiscussionSchema.ts

```diff
@@ -0,0 +1,43 @@
+import { IConfigurationSchema } from "../../core";
+import {
+  ENTITY_ACCESS_SCHEMA,
+  ENTITY_CATEGORIES_SCHEMA,
+  ENTITY_NAME_SCHEMA,
+  ENTITY_TAGS_SCHEMA,
+} from "../../core/schemas/shared";
+
+export type DiscussionEditorType = (typeof DiscussionEditorTypes)[number];
+export const DiscussionEditorTypes = ["hub:discussion:edit"] as const;
+
+/**
+ * defines the JSON schema for a Discussion's editable fields
+ */
+export const DiscussionSchema: IConfigurationSchema = {
+  required: ["name"],
+  type: "object",
+  properties: {
+    name: ENTITY_NAME_SCHEMA,
```

Comment:
[6880](https://devtopia.esri.com/dc/hub/issues/6880) doesn't list a name field, do we need this?